requirements.txt

```
@@ -1,2 +1,2 @@
 requests
-urllib3
+urllib3<2
```

```suggestion
urllib3>=2
```

How about instead of capping the version, we require >=2 and change this:

https://github.com/IBM/jupyter-tools/blob/747256d3a994ab36abfeba501f14cc05307facd9/scripts/hub-stress-test.py#L229

To use `allowed_methods`?

Are you OK with making and testing that change?

---

I agree that this would be even better. I'm getting back to you after making and testing that change